packages/mobile/src/components/details-tile/DetailsTile.tsx

```diff
@@ -347,6 +326,22 @@ export const DetailsTile = ({
         >
           {renderDetailLabels()}
         </View>
+        <View style={styles.descriptionContainer}>
+          {description ? (
+            <HyperLink
```

Comment:
we have a new core hyperlink that comes with the correct styling and clicking external link now!

Reply:
I don't have an thing that I'm currently looking at to test this, but will update when doing deprecated component pass

Reply:
word, and also fwiw i think its okay to keep deprecated component like text around if you want, cause i don't thing the new one has all the capabilities we want, just a headsup if it becomes difficult